src/middlewared/middlewared/api/base/server/ws_handler/rpc.py

```diff
@@ -177,20 +201,26 @@ def notify_unsubscribed(self, collection: str, error: Exception | None):
         params = {"collection": collection, "error": None}
         if error:
             if isinstance(error, ValidationErrors):
-                params["error"] = self.format_truenas_validation_error(error, errors=list(error))
+                params["error"] = self.format_truenas_validation_error(
+                    error, errors=list(error)
+                )
             elif isinstance(error, CallError):
-                params["error"] = self.format_truenas_error(error.errno, str(error), extra=error.extra)
+                params["error"] = self.format_truenas_error(
+                    error.errno, str(error), extra=error.extra
+                )
             else:
                 params["error"] = self.format_truenas_error(errno.EINVAL, str(error))
 
         self.send_notification("notify_unsubscribed", params)
 
     def send_notification(self, method, params):
-        self.send({
-            "jsonrpc": "2.0",
-            "method": method,
-            "params": params,
-        })
+        self.send(
+            {
+                "jsonrpc": "2.0",
+                "method": method,
+                "params": params,
+            }
+        )
```

Comment:
Why refactoring this?

Reply:
auto-formatted so that the entire file follows some sort of uniformity

Reply:
I'll remove the auto-formatting changes since I don't want to bike-shed over that and I really could care less about it haha.

Reply:
Okay, kept the changes to just removing `jsonschema`